utils/json-utils/src/main/scala/pl/touk/nussknacker/engine/util/json/JsonSchemaUtils.scala

```diff
@@ -0,0 +1,44 @@
+package pl.touk.nussknacker.engine.util.json
+
+import io.circe.{Json, JsonNumber, JsonObject}
+import org.json.{JSONArray, JSONObject, JSONTokener}
+import io.circe.Json._
+
+object JsonSchemaUtils {
+
+  import collection.JavaConverters._
+
+  private val CirceJsonFolder: Json.Folder[Object] = new Json.Folder[Object] {
+    def onNull: Object = JSONObject.NULL
+    def onBoolean(value: Boolean): Object = Predef.boolean2Boolean(value)
```

Comment:
I think it will be handled automatically?

Reply:
If I set ` def onBoolean(value: Boolean): Object = value` then IDEA throws:
`the result type of an implicit conversion must be more specific than Object
    def onBoolean(value: Boolean): Object = value
`

Reply:
I think we usually use `value : java.lang.Boolean` or `def onBoolean(value: Boolean): java.lang.Boolean = value` in such cases